subgraphs/notional-finance/networks.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Where does this file come from? iirc this may have been auto generated from the `messari-subgraph-cli`. If that is the case, just delete and update the cli bc that is no longer the case

Reply:
💯 

did upgrade my cli but this was generated long time ago.